backend/app/usecase/repository/user_changelog.go

```diff
@@ -6,6 +6,13 @@ import (
 	"github.com/short-d/short/app/entity"
 )
 
+// ErrEntryNotFound represents no entry found in the repository error
+type ErrEntryNotFound string
```

Comment:
@kapillamba4 As @alldroll suggested, `errors.Is` doesn't work in this case. I tried it myself. I think we need to do type check instead of using `errors.Is`.

@alldroll Want to retain some information for logging.

Reply:
@alldroll I think @kapillamba4  want to gracefully handle the error in the usecase. Please let me know if I am wrong.

Reply:
@byliuyang @alldroll 
I just read the article, maybe we can write our own custom Is method and check type there to see if it matches ErrEntryNotFound.
Section "Customizing error tests with Is and As methods" in  https://blog.golang.org/go1.13-errors or should we Wrap errors with %w ?

Reply:
Aha, I reviewed the article again. I meant `As`, not `Is`!!!

Reply:
Can we try As?

Reply:
Yes @byliuyang

Reply:
@kapillamba4 Great!

Reply:
Hi @alldroll, I think @kapillamba4 addressed your concern using `As`. Will merge now.